src/pages/index.js

```diff
@@ -49,30 +131,68 @@ const enableValidation = (settings) => {
 
 enableValidation(settings);
 
-// Section class Instance
+/*-----------------------------------------------------------------*/
+/*             Class Instances                          */
+/*-----------------------------------------------------------------*/
+
+// Section instance
 const cardSection = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (data) => {
-      cardSection.addItem(createCard(data));
+      cardSection.addItem(createCard(data), "append");
     },
   },
   cardContainer
 );
-cardSection.renderItems();
 
-// UserInfo class instance
-const userInfo = new UserInfo({
-  profileName: "#profile_name",
-  profileJob: "#profile_job",
+// Edit Avatar instance
+const popupWithAvatarForm = new PopupWithForm(
+  "#edit-avatar-modal",
+  (formData) => {
+    savingBtn.textContent = "Saving...";
+    const avatarLink = formData.avatar;
+    api
+      .changeProfilePic(avatarLink)
+      .then((data) => {
+        document.getElementById("profile__avatar").src = data.avatar;
+        popupWithAvatarForm.close();
+      })
+      .catch((err) => console.error(err));
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.